lib/topologies/topology_base.js

```diff
@@ -288,10 +282,19 @@ class TopologyBase extends EventEmitter {
     };
   }
 
+  // Sessions related methods
+  hasSessionSupport() {
+    return this.logicalSessionTimeoutMinutes != null;
+  }
+
   startSession(options) {
     return new ClientSession(this, this.s.sessionPool, options);
   }
 
+  endSessions(sessions, callback) {
```

Comment:
Is this moved for a reason?

Reply:
just to keep all the sessions-specific code localized, no functional reason